### ISSUE TITLE
Implement functionality of pressing a steering wheel button: Play/Pause

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#34]
## User story
1.	As a User
2.	I want to be able to press the {Play/Pause} button on the steering wheel
3.	So that I can simulate what happens when I press that button on a steering wheel in an actual car.

With this User story, the functionality of pressing the {Play/Pause} button on the steering wheel is to be implemented.
The button has two functionalities: play and pause.
When music is running and the button is pressed, the music should pause. When no music is running and the button is pressed, the music should play.
OnPressing the button, it should be visible that the button was pressed. E.g. by highlighting the button.

## Acceptance criteria
* The functionality of pressing the steering wheel button {Play/Pause} is implemented.
* When music is running and the button is pressed, the music should pause. 
* When no music is running and the button is pressed, the music should play.
* OnPressing the button, it is visible that the button was pressed.

## Notes
* 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
